src/components/Form.js

```diff
@@ -0,0 +1,326 @@
+
+
+const Form = (props) => {
+
+    return(
+        <form onSubmit={props.handleSubmit}>
+
+            <legend>So... You love podcasts and you want to discover more? Great! Here is the #1 app to get some recommendations. Enter your keyword, and click on Search Podcast button! You may select country or genre to help with the search results. Enjoy!</legend>
+
+            <label htmlFor="userKeyword"></label>
+            <input type="text" 
+            id="userKeyword" 
+            placeholder="Keyword" 
+            onChange={props.handleChange} 
+            aria-describedby="error" 
+            value={props.typedValue} required
+            />
+
+            <label htmlFor="countrySelection"></label>
+            <select name="countrySelection" id="countrySelection" onChange={props.handleCountry}>
+                {countryCodes.map((codes) => {
+                    return <option key={codes}>{codes}</option>
+                })}
+            </select>
+
+            <label htmlFor="genre"></label>
+            <select name="genre" id="genre" onChange={props.handleGenre}>
+                {genreCodes.map((genre)=>{
+                    return <option key={genre}>{genre}</option>
+                })}
+            </select>
+
```

Comment:
I'm not sure, but I think the selects should have the atribute value connect with an useState to be fully controlled by react... I couldn't follow the "props" whole path, but could be a good thing to look at.